src/renderer/inc/Cluster.hpp

```diff
@@ -29,11 +31,17 @@ namespace Microsoft::Console::Render
 
         const size_t GetColumns() const noexcept;
 
+        Cluster(const Cluster&);
+
+        Cluster& operator=(const Cluster&);
+
+        void SetColumns(size_t column);
+
     private:
         // This is the UTF-16 string of characters that form a particular drawing cluster
-        const std::wstring_view _text;

```

Comment:
I love the `const` here but I want to reuse existing `_text` and update the columns. So I had to make it this way.